doc-Managing_Infrastructure_and_Inventory/topics/Creating_a_VM_snapshot.adoc

```diff
@@ -6,13 +6,9 @@ To create a snapshot of a virtual machine:
 . Navigate to menu:Compute[Infrastructure > Virtual Machines].
 . Select the target virtual machine, and open it to view its details.
 . Under *Properties*, click *Snapshots* to show information about the virtual machine's snapshots.
-
-+
-image:Snapshot_info.png[]
-
-. Click image:plus_circle.png[](*Create a new snapshot for this VM*).
+. Click image:plus_green.png[](*Create a new snapshot for this VM*).
 . Enter snapshot details in *Description*.
-. (Optional) Check *Snapshot VM memory* for the snapshot to preserve the virtual machine's current runtime state.
+. (Optional) Check *Snapshot VM memory* for the snapshot to preserve the virtual machine's current runtime state. This option will appear only if the VM *Power State* is image:2143.png[](*On*). 
```

Comment:
@cbudz We use Select and Clear to refer to marking a check box per IBM style guide.